lib/rails_bump/checker/bundle_locally_check.rb

```diff
@@ -79,7 +79,9 @@ def gemfile_content
         GEMFILE
 
         @dependencies.each do |gem_name, gem_version|
-          result += "gem '#{gem_name}', '#{gem_version}'\n" unless gem_name == "rails"
+          gem_version = gem_version.include?(",") ? gem_version.split(", ").map { |i| "'#{i}'" }.join(", ") : gem_version
+
+          result += "gem '#{gem_name}', #{gem_version}\n" unless gem_name == "rails"
```

Comment:
can you please try this solution? @etagwerker 
```suggestion
          if gem_version.include?(",")
            gem_versions = gem_version.split(", ").map { |i| "'#{i}'" }.join(", ")
            result += "gem '#{gem_name}', #{gem_versions}\n" unless gem_name == "rails"
          else
            result += "gem '#{gem_name}', '#{gem_version}'\n" unless gem_name == "rails"
          end
```